packages/stryker-karma-runner/package.json

```diff
@@ -49,5 +49,10 @@
     "nicojs <jansennico@gmail.com>",
     "Simon de Lang <simondelang@gmail.com>",
     "Philipp Weissenbacher <philipp.weissenbacher@gmail.com>"
-  ]
+  ],
+  "initStrykerConfig": {
+    "karmaConfigFile": "karma.conf.js",
+    "files": null,
```

Comment:
We specified in the docs that this is mandatory. I understand that it will be filled at runtime but it's odd that it's `null` in the config file.